Tasks/Common/PowershellHelpers/Helpers.ps1

```diff
@@ -77,7 +77,7 @@ function Invoke-ActionWithRetries
             }
         }
 
-        if (!$RetryMessage)
+        if ($RetryMessage)
```

Comment:
Do you want to bump up task version or not needed?

Reply:
Task version in master is already bumped up in comparison to M137.